xarray/backends/zarr.py

```diff
@@ -128,14 +118,14 @@ def _open_zarr_group(store=None, overwrite=None, synchronizer=None,
 
 
 def _dask_chunks_to_zarr_chunks(chunks):
-    '''this function dask chunks syntax to zarr chunks'''
+    '''this function translates dask chunk syntax to zarr chunk syntax'''
     if chunks is None:
         return chunks
 
     all_chunks = product(*chunks)
     first_chunk = next(all_chunks)
     for this_chunk in all_chunks:
-        if not (this_chunk == first_chunk):
+        if this_chunk != first_chunk:
```

Comment:
I think this function may also need an update to handle the last chunk being smaller?

Reply:
Actually, it looks like this helper function `_dask_chunks_to_zarr_chunks` is never called -- can it simply be deleted instead?